src/mme-app/main.c

```diff
@@ -59,7 +59,7 @@ void
 check_mme_hdlr_status()
 {
 	/*TODO: not thread-safe*/
-	if(!(g_mme_hdlr_status ^ 511)) /*111111111 - 1 bit for init of 1 stage */
+	if(!(g_mme_hdlr_status ^ 1023)) /*11 1111 1111 - 1 bit for init of 1 stage */
```

Comment:
Can we do something better here. I tried to understand logic of 511. Would be good if we use bitmap for each thread and then do shifting. 
and here while comparing if we use 0x1FF instead of 511....

Reply:
This is using bitmap. Can change to hex for understanding.